### PR TITLE
fix: Support new json type url

### PR DESCRIPTION
**Pull request by johanandren**
Continuation of #2239 

For compatibility with runtime 1.2.0

Also bumps runtime to 1.2.0

### CODE REVIEW

File: project/Dependencies.scala [hunk 1]
```diff
@@ -10,7 +10,7 @@ object Dependencies {
     val RuntimeVersion = System.getProperty(
       "kalix-runtime.version",
       // temporarily accept the old system property name
-      System.getProperty("kalix-proxy.version", "1.1.46"))
+      System.getProperty("kalix-proxy.version", "1.2.0"))
```
Comment (efgpinto) [inline project/Dependencies.scala:13]: If we are bumping here, should we also bump the remaining places? Or maybe we do on a separate PR?

**Review by efgpinto — APPROVED** [review]
LGTM

File: project/Dependencies.scala (hunk 1 above)
Comment (johanandren) [reply project/Dependencies.scala:13]: Ah, I didn't notice that from Ciscos original PR, hmm. Might be fine to do here, to make sure tests are run against the new version. We anyway want that bump.

**Comment by efgpinto** [comment]
I jsut remembered we have some hardcoded ones [here](https://github.com/lightbend/kalix-jvm-sdk/blob/f46439ce1b7180caba9fe7cc7025248aa2e310de/sdk/java-sdk-protobuf-testkit/src/main/scala/kalix/javasdk/testkit/impl/EventingTestKitImpl.scala#L384)

**Comment by johanandren** [comment]
I think that's fine in that direction, the important part here is to accept the json.akka.io typeurl when it shows up
